packages/embeds/embed-core/src/embed-iframe.ts

```diff
@@ -341,15 +341,15 @@ function keepParentInformedAboutDimensionChanges() {
     }
 
     const mainElementStyles = getComputedStyle(mainElement);
-    const contentHeight =
-      mainElement.offsetHeight +
-      parseInt(mainElementStyles.marginTop) +
-      parseInt(mainElementStyles.marginBottom);
-    const contentWidth =
-      mainElement.offsetWidth +
-      parseInt(mainElementStyles.marginLeft) +
-      parseInt(mainElementStyles.marginRight);
-
+    const contentHeight = Math.ceil(
```

Comment:
```suggestion
    // Use, .height as that gives more accurate value in floating point. Also, do a ceil on the total sum so that whatever happens there is enough iframe size to avoid scroll.
    const contentHeight = Math.ceil(
```